sympy/sets/tests/test_sets.py

```diff
@@ -453,6 +453,12 @@ def test_contains():
     assert RootOf(x**5 + x**3 + 1, 0) in S.Reals
     assert not RootOf(x**5 + x**3 + 1, 1) in S.Reals
 
+    # non-bool results
+    assert Union(Interval(1, 2), Interval(3, 4)).contains(x) == \
```

Comment:
No, they just establish that no change should lose these elements by, perhaps, just testing for true and false values: one must account for the Basic case.